templates.json

```diff
@@ -63,7 +63,7 @@
                 "debuggerConsole": null,
                 "mainFile": "Program.cs",
                 "mainFileBreakpointLine": 29,
-                "successDebugMessage": "Loaded 'Anonymously Hosted DynamicMethods Assembly'."
+                "successDebugMessage": "System.Linq.Expressions.dll'. Skipped loading symbols. Module is optimized and the debugger option 'Just My Code' is enabled."
```

Comment:
Maybe it's better to add some `Console.Writeline("Hello world")` to the template to be sure that this will not be changed by external libs ...

Reply:
Ok, I will also add a "Hello World" on the cppQML template for the same purpose as this one.